README.md

```diff
@@ -259,39 +267,13 @@ REST API to push the `tofino-netcfg.json` file.
 
 Check the ONOS log for potential errors.
 
-## Using Trellis with Stratum+Tofino switches
-
-Check the official Trellis documentation here:
-<https://docs.trellisfabric.org>
-
-In the "Device Configuration" section:
-<https://docs.trellisfabric.org/configuration/device-config.html>
-
-make sure to replace the `basic` JSON node for OpenFlow devices with the one
-provided in `tofino-netcfg.json`, for example:
-
-```json
-{
-  "devices" : {
-    "device:leaf-1" : {
-      "segmentrouting" : {
-        "ipv4NodeSid" : 101,
-        "ipv4Loopback" : "192.168.0.201",
-        "ipv6NodeSid" : 111,
-        "ipv6Loopback" : "2000::c0a8:0201",
-        "routerMac" : "00:00:00:00:02:01",
-        "isEdgeRouter" : true,
-        "adjacencySids" : []
-      },
-      "basic": {
-        "managementAddress": "grpc://10.0.0.1:28000?device_id=1",
-        "driver": "stratum-tofino",
-        "pipeconf": "org.stratumproject.fabric.montara_sde_9_7_0"
-      }
-    }
-  }
-}
-```
+## Using SD-Fabric with Stratum+Tofino switches
+
+Check the official SD-Fabric documentation here:
+<https://docs.sd-fabric.org>
+
+In the "Network Configuration" section:
+<https://docs.sd-fabric.org/master/configuration/network.html>
```

Comment:
@charlesmcchan @ccascone do you think it is a good idea to use the SD-Fabric doc instead of the Trellis one?

Reply:
Totally